### PR TITLE
Add pyodbc-sqlcmd image

### PR DESCRIPTION
**Pull request by cham11ng**
Add pyodbc-sqlcmd image which extends the laudio/pyodbc image and includes `sqlcmd`

### CODE REVIEW

File: images/pyodbc-sqlcmd/test/test_connection.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+def test_pyodbc_pg():
+    ''' Test connection to PostgreSQL Server. '''
+    result = exec_pyodbc_query(PG, "SELECT 'It works!'")
+
+    assert result == 'It works!'
+
+
+def test_sqlcmd():
+    ''' Test connection to MSSQL Server. '''
```
Comment (kabirbaidhya) [inline images/pyodbc-sqlcmd/test/test_connection.py]:
```suggestion
def test_sqlcmd_mssql():
    ''' Test connection to MSSQL Server through sqlcmd. '''
```

**Comment by kabirbaidhya** [comment]
@cham11ng Failing build.

**Comment by kabirbaidhya** [comment]
Also do update the README about using this image as an alternative somewhere.

File: README.md [hunk 2]
```diff
@@ -89,6 +89,12 @@ Examples illustrating data pipelines using pyodbc.
 2. [Data Transfer Example (MSSQL)](examples/data-transfer-mssql)
 3. [Data Transfer Example (MySQL)](examples/data-transfer-mysql)
 
+### Scripts
```
Comment (kabirbaidhya) [inline README.md]: Images

File: images/pyodbc-sqlcmd/test/test_connection.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    result = exec_pyodbc_query(PG, "SELECT 'It works!'")
+
+    assert result == 'It works!'
+
+
+def test_sqlcmd_mssql():
+    ''' Test connection to MSSQL Server. '''
+    result = exec_sqlcmd_query("SELECT 'It works!'")
+
+    assert result == b'         \n---------\nIt works!\n\n(1 rows affected)\n'
```
Comment (kabirbaidhya) [inline images/pyodbc-sqlcmd/test/test_connection.py]: Don't have to make it very specific - it's brittle. You can simply do:
```
assert "It works!" in result
```

File: .travis.yml [hunk 4]
```diff
@@ -24,7 +24,7 @@ install:
   - hub --version
 
   # pyright
-  - npm install -g pyright
+  - npm install -g pyright@1.1.41
```
Comment (cham11ng) [inline .travis.yml:27]: Had to do this. (https://github.com/microsoft/pyright/issues/769)

```
Cannot find module 'worker_threads'
```

Comment (kabirbaidhya) [reply .travis.yml:27]: OK.

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**